.github/workflows/sanity/setup-apache.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# Get vars
+WORKSPACE=$1
+PHP_VERSION=$2
+
+# Install apache
+apt-get update && apt-get install apache2 libapache2-mod-php$PHP_VERSION
```

Comment:
```suggestion
apt update && apt install -y apache2 libapache2-mod-php$PHP_VERSION
```
I don't know how it can work without the `-y` :thinking: